docs/buildah.md

```diff
@@ -58,6 +58,7 @@ Print the version
 | buildah-config(1)     | Update image configuration settings.                                                                 |
 | buildah-containers(1) | List the working containers and their base images.                                                   |
 | buildah-copy(1)       | Copies the contents of a file, URL, or directory into a container's working directory.               |
+| buildah-export(1)     | Export the contents of a container's filesystem as a tar archive                                     |
```

Comment:
Doh!  Yep, you're right I'd the wrong file.  It would be a nice add here too though.